### PR TITLE
[Bombastic Perks] Add more perks

### PR DESCRIPTION
**Pull request by Standing-Storm**
#### Summary
Mods "[Bombastic Perks] Add more perks"

#### Purpose of change

Had more perk ideas

#### Describe the solution

Currently four perks. That's a good place to stop.

Two regular perks:

- Perfect Timing: You used to annoy your friends before the Cataclysm by always being *exactly* on time and you've maintained that characteristic to this day.  You always know what time it is and can set alarms even without a watch or a phone.
- Bedtime Stories: You've always loved reading, especially before you fall asleep, and have gotten very good at tearing through books as a way to wind down.  You read books 50% faster, but only when you read them between 8 p.m. and 1 a.m. Condition: Cannot have _Hates Books_ or _Illiterate_

And two playstyle perks:

- Ascetic Empowerment: People talk about getting a high from fasting and for you that's true on a more visceral level.  You gain bonuses to all stats that scale with how hungry, thirsty, and tired you are.
- Compressed Eating Window: You're so used to squeezing all your meals into a short time that your body has adapted to it.  You gain +150% kcal and vitamins from any meals eaten between noon and 6 p.m., but 0 calories and vitamins for any meals eaten outside that time period.

#### Testing

Ascetic Empowerment:
image

Other perks also work properly.

### CODE REVIEW

File: data/mods/BombasticPerks/perkmenu.json [hunk 1]
```diff
@@ -708,6 +708,42 @@
...
+        "condition": { "not": { "u_has_trait": "perk_perfect_time" } },
+        "text": "Gain [<trait_name:perk_perfect_time>]",
+        "effect": [
+          { "set_string_var": "<trait_name:perk_perfect_time>", "target_var": { "context_val": "trait_name" } },
+          {
+            "set_string_var": "<trait_description:perk_perfect_time>",
+            "target_var": { "context_val": "trait_description" }
+          },
+          { "set_string_var": "perk_perfect_time", "target_var": { "context_val": "trait_id" } },
+          { "set_string_var": "No Requirements", "target_var": { "context_val": "trait_requirement_description" } },
```
Comment (Qrox) [inline data/mods/BombasticPerks/perkmenu.json:721]:
```suggestion
          { "set_string_var": "No Requirements", "target_var": { "context_val": "trait_requirement_description" }, "i18n": true },
```

File: data/mods/BombasticPerks/perkmenu.json [hunk 2]
```diff
@@ -708,6 +708,42 @@
...
+        "effect": [
+          { "set_string_var": "<trait_name:perk_bedtime_reader>", "target_var": { "context_val": "trait_name" } },
+          {
+            "set_string_var": "<trait_description:perk_bedtime_reader>",
+            "target_var": { "context_val": "trait_description" }
+          },
+          { "set_string_var": "perk_bedtime_reader", "target_var": { "context_val": "trait_id" } },
+          {
+            "set_string_var": "Cannot have the Hates Books or Illiterate traits",
+            "target_var": { "context_val": "trait_requirement_description" }
```
Comment (Qrox) [inline data/mods/BombasticPerks/perkmenu.json:738]:
```suggestion
            "target_var": { "context_val": "trait_requirement_description" },
            "i18n": true
```

**Comment by GuardianDll** [comment]
Why tho?

**Comment by Standing-Storm** [comment]
> Why tho?

Because I thought it was already merged. -_-

**Pull request merged**